<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleInstructionSetPage</class>
 <widget class="QWidget" name="VehicleInstructionSetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="VehicleInstructionSetPage_Button_EB">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>紧急广播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleInstructionSetPage_Button_FBRMC2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>强迫制动缓解Mc2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleInstructionSetPage_Button_FBRMC1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>强迫制动缓解Mc1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleInstructionSetPage_Button_EBStop">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>紧急广播停止</string>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleInstructionSetPage_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>440</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleInstructionSetPage_Button_forcewind">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>强迫泵风</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleInstructionSetPage_Button_quit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>340</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleInstructionSetPage_Button_Sandforbidden">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>撒沙禁止</string>
   </property>
  </widget>
  <zorder>VehicleInstructionSetPage_background</zorder>
  <zorder>VehicleInstructionSetPage_Button_EB</zorder>
  <zorder>VehicleInstructionSetPage_Button_FBRMC2</zorder>
  <zorder>VehicleInstructionSetPage_Button_FBRMC1</zorder>
  <zorder>VehicleInstructionSetPage_Button_EBStop</zorder>
  <zorder>VehicleInstructionSetPage_Button_forcewind</zorder>
  <zorder>VehicleInstructionSetPage_Button_quit</zorder>
  <zorder>VehicleInstructionSetPage_Button_Sandforbidden</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
